src/pages/[postcode]/material/RecycleAtHome.tsx

```diff
@@ -0,0 +1,161 @@
+import { useTranslation, Trans } from 'react-i18next';
+import { Link, useParams } from 'react-router-dom';
+import '@etchteam/diamond-ui/canvas/Card/Card';
+import '@etchteam/diamond-ui/composition/Grid/Grid';
+import '@etchteam/diamond-ui/composition/Grid/GridItem';
+import '@etchteam/diamond-ui/control/Button/Button';
+
+import '@/components/canvas/IconCircle/IconCircle';
+import '@/components/content/Icon/Icon';
+import '@/components/composition/BorderedList/BorderedList';
+import '@/components/composition/Container/Container';
+import '@/components/control/IconLink/IconLink';
+import '@/components/content/ContainerSvg/ContainerSvg';
+
+import containerName from '@/lib/containerName';
+import { DryScheme } from '@/types/locatorApi';
+
+function ManySchemes({
+  schemes,
+  schemesWithContainers,
+}: {
+  readonly schemes: DryScheme[];
+  readonly schemesWithContainers: DryScheme[];
+}) {
+  const { postcode } = useParams();
+  const tContext = 'material.recycleAtHome.manySchemes';
+  const allSchemesRecycle = schemesWithContainers.length === schemes.length;
+  const sortedSchemes = schemes.toSorted((scheme) => {
+    return scheme.containers.length > 0 ? -1 : 1;
+  });
+
+  return (
+    <>
+      <p className="text-size-sm">
+        <Trans
+          i18nKey={`${tContext}.collection${allSchemesRecycle ? 'All' : 'Some'}`}
+          components={{ bold: <strong /> }}
+        />
+      </p>
+      <ul role="list" className="list-style-none diamond-spacing-bottom-md">
+        {sortedSchemes.map((scheme) => {
+          const safeSchemeName = encodeURIComponent(scheme.name);
+          const recyclable = scheme.containers.length > 0;
+
+          return (
+            <li key={scheme.name} className="diamond-spacing-bottom-sm">
+              <locator-icon-link>
+                <Link
+                  to={`/${postcode}/home/collection?scheme=${safeSchemeName}`}
```

Comment:
Do we handle 404s on this route? Scheme names can change.

Reply:
The scheme name is being used for the york issue where the LA can have a few schemes.

The route doesn't exist yet, but when it does, If the scheme name isn't provided or isn't found it'll show the first scheme the LA has.

If the API provided an id I could use that instead.

Reply:
I think scheme names are probably more resilient than IDs, the ID will change at least yearly, whereas the scheme name doesn't change as often (had a dig through the data).